src/fondant/core/schema.py

```diff
@@ -197,7 +197,13 @@ def from_dict(cls, json_schema: dict):
             fields = [(name, cls.from_dict(prop)) for name, prop in properties.items()]
             return cls.struct(fields)
 
+        if type_name == "timestamp":
+            return cls(type_name)
```

Comment:
I don't think we need to support this as a type, only as a format, right?

Reply:
Good catch, leftover from when I was trying something different. Removed